From 0023179ac521c7f25e1b4a5b215000ab5aa0a859 Mon Sep 17 00:00:00 2001
From: Karel Balej <balejk@matfyz.cz>
Date: Thu, 23 Jan 2025 18:04:00 +0100
Subject: [PATCH 1/6] config: remove hide_scrollbars from the default rc.lua

Hiding the scrollbars causes rendering issues with smooth scrolling --
in particular, the scrolling doesn't often take effect until some other
rendering is forced, such as changing a link color by hovering over it,
invoking the follow hints or selecting some text which causes it to get
highlighted.

Other approaches to hide the bars, such as using "display: none"
instead, lead to the same problem. It is possible that it will be
possible to restore this eventually using the standardized scrollbar
properties (scrollbar-width) provided they won't have the same
shortcoming as this WebKit-specific one.

Fixes: https://github.com/luakit/luakit/issues/1007
Link: https://github.com/luakit/luakit/issues/1081#issuecomment-2081564587
---
 CHANGELOG.md  | 2 ++
 config/rc.lua | 3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/config/rc.lua b/config/rc.lua
index f4d1f058..a9873b07 100644
--- a/config/rc.lua
+++ b/config/rc.lua
@@ -178,9 +178,6 @@ local error_page = require "error_page"
 -- Add userstyles loader
 local styles = require "styles"
 
--- Hide scrollbars on all pages
-local hide_scrollbars = require "hide_scrollbars"
-
 -- Add a stylesheet when showing images
 local image_css = require "image_css"
 
-- 
2.45.3

